.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,8 +13,9 @@ Oh no, a bug! It happens. Thanks for reporting an issue with NewPipe. To make it
 <!-- The comments between these brackets won't show up in the submitted issue (as you can see in the preview). -->
 
 ### Checklist
+<!-- The first box has been checked for you to show you how it is done. -->
 
-- [ ] I am using the latest version. <!-- compare Releases page and your version given in About in the app drawer -->
+- [x] I am using the latest version. <!-- compare Releases page and your version given in About in the app drawer -->
```

Comment:
I don't know if pre-checking one of the boxes is the best idea, as people might then skip that item

Reply:
This is the least problematic of the items. People generally do ensure they are on the latest version because it is easy to check. But checking the repo for duplicates or reading the contribution document takes time.